Core/GDCore/Extensions/Builtin/WindowExtension.cpp

```diff
@@ -38,6 +38,17 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsWindowExtension(
                     true)
       .SetDefaultValue("yes");
 
+  extension
+      .AddCondition(
+          "IsFullScreen",
+          _("Fullscreen activated?"),
+          _("Check if the game is currently in fullscreen."),
+          _("Is the game in fullscreen?"),
```

Comment:
Usually we use an affirmative sentence: "The game is in fullscreen"

Reply:
This terminology makes sense for many of them. "The game is maximized", "The game is in fullscreen", "The game is in focus", etc.